### DISCUSSION TITLE
Did the JSON schema change to now require Minimum & Maximum encoded values for every standard_charges object?

### DISCUSSION THREAD

**Discussion by knh762002** [CMSgov/hospital-price-transparency#88] [Q&A] [answered]
Are Minimum and Maximum encoded values now required for every standard_charges object even if they do not contain payers_information objects?  As of last week my JSON files were validating, but now they are erroring on standard_charges objects that do not contain Minimum and Maximum encoded values.  I have CDM code types that do not have payer negotiated rates, therefore will not have a Minimum and Maximum.  Am I now supposed to encode the gross_charge amount as both the Minimum and Maximum to get the JSON to validate?  The CSV file that contains the same encoded values validates without an issue.

**This Object Errors:**

"standard_charge_information": [
{
"description": "CDM ITEM CODE DESCRIPTION",
"code_information": [
{
"code": "5555555555",
"type": "CDM"
}
],
"standard_charges": [
{
"gross_charge": 2434.25,
"discounted_cash": 2068.88,				
"setting": "both",
"billing_class": "facility"
}
]
}
]

**This Object Validates:**

"standard_charge_information": [
{
"description": "CDM ITEM CODE DESCRIPTION",
"code_information": [
{
"code": "5555555555",
"type": "CDM"
}
],
"standard_charges": [
{
**"minimum": 2434.25,
"maximum": 2434.25,**
"gross_charge": 2434.25,
"discounted_cash": 2068.88,				
"setting": "both",
"billing_class": "facility"
}
]
}
]

![image](https://github.com/CMSgov/hospital-price-transparency/assets/152221202/80bf816d-1b20-45e2-a15d-251f6437839c)

**Comment by shaselton-usds** [comment, marked as answer]
Hi @knh762002 

Thanks for reaching out! We've introduced a fix to the validator that can be found over here: https://github.com/CMSgov/hpt-validator/pull/32

Please try your file again under the same circumstances to see if you are still running into the issue.
